docs/siteconfig.rst

```diff
@@ -242,3 +242,15 @@ Here is a sample configuration with many of the options set and documented::
     pelagos:
       endpoint: http://head.ses.suse.de:5000/
       machine_types: ['type1', 'type2', 'type3']
+
+    # Defines the bases for container images
+    container_images:
+      # This is a read-only mirror to limit the number of requests for downloading
+      # prometheus, grafana, node-exporter and alertmanager. Typically, this is a mirror
+      # for docker.io, but could be a mirror for any search registry, as the images are
+      # referenced without an explicit registry name.
+      base_registry_mirror: 'vossi04.front.sepia.ceph.com:5000'
```

Comment:
Just `registry`?

Reply:
* it's not a registry, but a mirror of a registry
* it can also contain a port

Reply:
1. mirror is an equivalent of registry, if there is a real distinction, name it `registry_mirror_host`.
2. you can see another example of `_host` in the config which also have ports and protocol, it is used for concise notation, we may follow the style of teuthology config I guess.

Reply:
could you explain why `base_` prefix required?

Reply:
> could you explain why `base_` prefix required?

Right. it should be changed into a dict

```yaml
container_images:
  registry_mirrors:
    docker.io: 'vossi04.front.sepia.ceph.com:5000'
```